.buildkite/windows-docker.ps1

```diff
@@ -0,0 +1,6 @@
+$dockerUrl="https://download.docker.com/win/static/stable/x86_64/docker-17.09.0-ce.zip"
```

Comment:
is there something that only contains the client, but not the server?

Reply:
No they always contain both

Reply:
could you add a comment on where this script comes from?

I am guessing this is copied from somewhere.

otherwise, it would be easier to write a linux-like bash script (which will work in this context), rather than using a ps1 script.